cloud_utils/cache/stores/redis.py

```diff
@@ -52,15 +52,17 @@ async def get_item(key: str):
             raise KeyError
 
     async def set_item(key: str, value):
+        ttl_function = gamla.just(ttl) if type(ttl) == int else ttl
```

Comment:
You don't need this. Set the type to be Callable and send this function from `make_store`